src/applications/disability-benefits/all-claims/tests/pages/secondaryOtherSourcesHelp.unit.spec.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import { DefinitionTester } from '../../../../../platform/testing/unit/schemaform-utils.jsx';
+import { mount } from 'enzyme';
+import formConfig from '../../config/form';
+
+describe('Add secondary other sources of information help ', () => {
+  const {
+    schema,
+    uiSchema,
+  } = formConfig.chapters.disabilities.pages.secondaryOtherSourcesHelp0;
+
+  it('should render', () => {
+    const form = mount(
+      <DefinitionTester
+        definitions={formConfig}
+        schema={schema}
+        uiSchema={uiSchema}
+        data={{
+          'view:selectablePtsdTypes': {
+            'view:combatPtsdType': true,
+          },
+          otherSources0: 'yes',
```

Comment:
:thinking: Should this be `incident0: { otherSources: true } }` now? Looking at [this line](https://github.com/department-of-veterans-affairs/vets-website/pull/9221/files#diff-46581ec6a5a0c4e644781b1456042c61R15).

Although...maybe this doesn't matter after all if the page is rendering. I think the `DefinitionTester` doesn't check the `depends` callback and instead always renders the page.

You could probably get away with `data={{}}`. :man_shrugging: